.konflux/Containerfile.plugin

```diff
@@ -0,0 +1,31 @@
+# Build Stage
+FROM  brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.22 AS builder
+WORKDIR /go/src
+COPY . /go/src
+RUN GIT_COMMIT=$(git rev-parse HEAD) && \
+  CGO_ENABLED=0 GOOS=linux go build -a -mod=readonly \
+  -ldflags "-X github.com/redhat-developer/gitops-backend/pkg/health.GitRevision=${GIT_COMMIT}" ./cmd/backend-http
+
+# Final Stage
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+WORKDIR /
+COPY --from=builder /go/src/backend-http .
+EXPOSE 8080
+ENTRYPOINT ["./backend-http"]
+
+LABEL \
+    name="openshift-gitops-1/gitops-rhel8" \
+    version=${CI_CONTAINER_VERSION} \
+    License="Apache 2.0" \
+    com.redhat.component="openshift-gitops-container" \
+    com.redhat.delivery.appregistry="false" \
+    release=${CI_CONTAINER_RELEASE} \
+    upstream-version=${CI_UPSTREAM_VERSION} \
+    upstream-vcs-ref="${CI_GITOPS_BACKEND_UPSTREAM_COMMIT}" \
```

Comment:
I think we need to find replacements for `CI_*` env vars. If I am not wrong, these vars are CPAAS specific. We need to figure out konflux equivalent for these or manually define them.

Reply:
Thanks @svghadi , you are right, I will try to find konflux equivalent for these

Reply:
Even I need them for dex builds. Let me know if you find them.

Reply:
We can address this during release pipeline configurations, shouldn't be a blocker for merging this PR.